parmed/openmm/parameters.py

```diff
@@ -178,10 +179,16 @@ def _write_omm_atom_types(self, dest):
         dest.write(' </AtomTypes>\n')
 
     def _write_omm_residues(self, dest):
+        if not self.residues: return
         dest.write(' <Residues>\n')
         for name, residue in iteritems(self.residues):
             if not isinstance(residue, ResidueTemplate):
                 continue
+            # if AtomTypes are available, this will throw an exception if the
+            # residue is claiming a Type that doesn't exist
+            if self.atom_types:
```

Comment:
Isn't this what `typeify_templates` does?  Why did you comment-out the call above, only to essentially re-implement it here?